NOTES.WIN

```diff
@@ -147,27 +185,28 @@
 
  If you link with OpenSSL .DLLs, then you're expected to include into
  your application code small "shim" snippet, which provides glue between
- OpenSSL BIO layer and your compiler run-time. See the OPENSSL_Applink
- manual page for further details.
+ OpenSSL BIO layer and your compiler run-time.
```

Comment:
...include into your application *a* small "shim" snippet, which provides glue between *the* OpenSSL BIO layer and your compiler run-time.

Reply:
Done